### PR TITLE
defaults: add Decibels & Music

### PR DESCRIPTION
**Pull request by cassidyjames**
https://phabricator.endlessm.com/T32696

### CODE REVIEW

**Review by wjt — CHANGES_REQUESTED** [review]
I don't think these should be mandatory, I think they should be in `apps_add` (i.e. not present in the `base` image), as Rhythmbox is.

And, presumably you intended to remove Rhythmbox from the defaults?

**Comment by cassidyjames** [comment]
@wjt ah, I saw Totem there and thought Decibels served a similar purpose; without it, I guess audio files will just open in Totem? I agree Music should replace Rhythmbox in `apps_add`, though; my mistake.

File: config/defaults.ini [hunk 1]
```diff
@@ -502,7 +503,7 @@ apps_add =
   org.gimp.GIMP
   org.gnome.Cheese
   org.gnome.Gnote
-  org.gnome.Rhythmbox3
+  org.gnome.Music
```
Comment (wjt) [inline config/defaults.ini:506]: Very convenient that they fall in the same place in the sort order!

File: config/defaults.ini [hunk 2]
```diff
@@ -466,6 +466,7 @@ apps_add_mandatory =
 [flatpak-remote-flathub]
 repo_file = https://dl.flathub.org/repo/flathub.flatpakrepo
 apps_add_mandatory =
+  com.vixalien.decibels
```
Comment (wjt) [inline config/defaults.ini:469]: Yes, on second thoughts, your line of reasoning makes sense: being able to play audio files is essential functionality, so decibels should go here; while having a music library app is not.

**Review by wjt — APPROVED** [review]

File: config/defaults.ini (hunk 1 above)
Comment (cassidyjames) [reply config/defaults.ini:506]: Indeed!

**Pull request merged**